pkg/features/embedded/elemental-rootfs/usr/lib/dracut/modules.d/30elemental-rootfs/module-setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# called by dracut
+check() {
+    require_binaries "$systemdutildir"/systemd || return 1
+    return 255
+}
+
+# called by dracut 
+depends() {
+    echo systemd rootfs-block dm fs-lib
+    return 0
+}
+
+# called by dracut
+installkernel() {
+    instmods overlay
+}
+
+# called by dracut
+install() {
+    declare moddir=${moddir}
+    declare systemdutildir=${systemdutildir}
+    declare systemdsystemunitdir=${systemdsystemunitdir}
+
+    inst_multiple \
+        mount mountpoint sort rmdir findmnt rsync cut realpath basename lsblk
```

Comment:
same as below, this list feels outdated. Be aware that these might be required to be included to some other module then.